### DISCUSSION TITLE
Resend confirmation mail

### DISCUSSION THREAD

**Discussion by RonnyWinkler** [hizzle-co/noptin#874] [Ideas]
Some users never confirmed their email address with doule opt-in. Perhaps they just overlook the mail in their inbox,
Is is somehow possible to send the confirmation mail again?
Simple just from subscriber list? Filter all not yet confirmed users, mark all, Resend confirmation.
Or just with a button in subscriber details.

**Comment by picocodes** [comment]
Hi Ronny,

We'll add this in Noptin 3.4.6 scheduled to be released next week.

Thanks.
